iaso/api/entity.py

```diff
@@ -173,6 +175,14 @@ def get_queryset(self):
             queryset = queryset.filter(attributes__created_by_id=created_by_id)
         if created_by_team_id:
             queryset = queryset.filter(attributes__created_by__teams__id=created_by_team_id)
+        if groups:
+            if isinstance(groups, str):
+                group_ids = groups.split(",")
+            elif isinstance(groups, int):
+                group_ids = [groups]
+            else:
+                group_ids = groups
+            queryset = queryset.filter(attributes__org_unit__groups__in=group_ids)
```

Comment:
I think queryset = queryset.filter(attributes__org_unit__groups__in=group_ids.split(",")) can work too like on the line 157

Reply:
You're right! Not sure how I got to this "safer" version, but it seems overkill. I'm changing it :+1: